### PR TITLE
Stage bundle 4.6

### PR DESCRIPTION
**Pull request by petr-balogh**
Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Comment by petr-balogh** [comment]
Running verification here:  https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/12978/console

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -904,6 +905,22 @@ def create_catalog_source(image=None, ignore_upgrade=False):
...
     if not image:
         image = config.DEPLOYMENT.get('ocs_registry_image', '')
+    if config.DEPLOYMENT.get('stage_rh_osbs'):
+        image = config.DEPLOYMENT.get(
+            "stage_index_image", constants.OSBS_BOUNDLE_IMAGE
+        )
+        osbs_image_tag = config.DEPLOYMENT.get(
+            "stage_index_image_tag", f"v{get_ocp_version()}"
+        )
+        image += f":{osbs_image_tag}"
```
Comment (vavuthu) [inline ocs_ci/deployment/deployment.py:915]: Didn't see the usage of "image" in if block and we are overriding the image in https://github.com/red-hat-storage/ocs-ci/pull/3039/files#diff-153ccf5916ca230718b5455c349afbcdR929

Comment (petr-balogh) [reply ocs_ci/deployment/deployment.py:915]: It's used.

```
image_and_tag = image.split(':')
```

So if we have the image there then it's continue with tag passed and it's not overwritten.

I've tested it and see that from the job I've used for testing the proper image:tag was used as intended.

Comment (vavuthu) [reply ocs_ci/deployment/deployment.py:915]: ahh, missed that

**Review by vavuthu — DISMISSED** [review]

**Review by vavuthu — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**